### ISSUE TITLE
request for command line option to set filename when using mate-screenshot

### ISSUE THREAD

**Issue by steve-1776** [mate-desktop/mate-utils#150]
Yes I know there are no feature requests allowed, but I can't write my own at present. Working on acquiring the skillset. In the meantime, it would be ideal if one could use a command line option ( -f is used with the corresponding gnome-screenshot) when using mate-screenshot to name the resulting file. It is disruptive to the workflow to have to use the dialog box to name the screenshot. Thank you.

**Comment by adefaria** [comment]
Yeah, again, like the old gnome-screenshot used to offer. Indeed, mate-screenshot's man page says it saves the screenshot to a file but not if you do mate-screenshot -a -c (yes Elizabeth, mate-screenshot honors -c). If you try to capture an area instead of the screen or a window you'll get no file when you are done!
